### PR TITLE
Even MORE trig doc movement...

### PR DESCRIPTION
**Pull request by kshyatt**
and I fixed a failing doctest in the manual, so doctests pass.

### CODE REVIEW

File: base/special/trig.jl [hunk 1]
```diff
@@ -319,10 +319,13 @@ for (finv, f) in ((:sec, :cos), (:csc, :sin), (:cot, :tan),
...
-for (fa, fainv) in ((:asec, :acos), (:acsc, :asin), (:acot, :atan),
-                    (:asech, :acosh), (:acsch, :asinh), (:acoth, :atanh))
+for (fa, fainv, fn) in ((:asec, :acos, "secant"), (:acsc, :asin, "cosecant"), (:acot, :atan, "cotangent"),
+                        (:asech, :acosh, "hyperbolic secant"), (:acsch, :asinh, "hyperbolic cosecant"), (:acoth, :atanh, "hyperbolic cotangent"))
+    name = string(fa)
     @eval begin
-        ($fa){T<:Number}(y::T) = ($fainv)(one(T) / y)
+        @doc """
+            $($name)(x)
+        Compute the $($fn) of `x`, where the output is in radians. """ ($fa){T<:Number}(y::T) = ($fainv)(one(T) / y)
```
Comment (tkelman) [inline base/special/trig.jl]: Did you run `make docs` on this? These are inverses. Is the output in radians for the hyperbolic ones?

Comment (Evizero) [inline base/special/trig.jl]: Just by comparing the old doc strings to the new it looks like you forgot the word "inverse" before the function name

Comment (kshyatt) [inline base/special/trig.jl]: This is what happens when you write PRs on a delayed train

Comment (tkelman) [inline base/special/trig.jl]: Santa Barbara, the only place west of the Mississippi where Amtrak is the *more* convenient option?

Comment (kshyatt) [inline base/special/trig.jl]: I don't have a car and the San Diego airport is a pain to get to from the university. Free WiFi + comfy chair = so may doc PRs

**Pull request merged**